### ISSUE TITLE
Frequent crash of Blender Launcher

### ISSUE THREAD

**Issue by L0Lock** [DotBow/Blender-Launcher#110]
**Describe the bug**
It happens quite often that Blender Launcher's tray icon is not there when I need it. It does start with Windows, but at some point that I can't identify, it just stops. I think it never happened to not have it the first time I want to use it after a startup.

**System (please complete the following information):**
 - Operation system: Windows 11 and 10
 - Blender Launcher version: 1.14.1
 - Log file: [BL.log](https://github.com/DotBow/Blender-Launcher/files/7499471/BL.log)

**Comment by DotBow** [comment]
Hi!
How often do you run blender via BL? Do you use multiple instances?

**Comment by L0Lock** [comment]
It can run from once a day to several times per hour depending on what I am doing. I do run multiple instances sometimes.

Unless I open Blender by running a .blend file, I always use BL.

**Comment by DotBow** [comment]
> 
> 
> It can run from once a day to several times per hour depending on what I am doing. I do run multiple instances sometimes.
> 
> Unless I open Blender by running a .blend file, I always use BL.
> 
> EDIT: Oh well... I ran Blender a few times when answering here, then closed everything. BL was running back then.
> 
> But now, about an hour later, I want to open Blender, and BL is not there... Here's the log with today's content: [2021-11-10 -- BL.log](https://github.com/DotBow/Blender-Launcher/files/7516397/2021-11-10.--.BL.log)

Thanks for logs, I will try to find what is broken as have time.

**Comment by guategeek** [comment]
This is happening for me as well, it just dies on me all the time.

I just found Blender Launcher today and was working on setting it up so I haven't used it much, but it doesn't stay open for long

Here is my log file
[BL.log](https://github.com/DotBow/Blender-Launcher/files/7566939/BL.log)

**Comment by DotBow** [comment]
Could you please test this build, I disabled instances counter to make sure that the issue in this thread.

https://drive.google.com/file/d/1vOPW0hbODzKRQzKVUgb7bYqxVy-u8q3-/view?usp=sharing

**Comment by TarnishElias** [comment]
Just want to chime in to say that I also have this exact issue.

**Comment by guategeek** [comment]
I have been running the build from the drive link you posted, and I haven't noticed it closing on me, if it is doing it its definitely not nearly as often.

**Comment by TarnishElias** [comment]
I've been running that linked dev version as well, but for me, even in just the few hours I've been using it, it already did the sudden closing at least 3 times, so there doesn't seem to be much of a difference.

**Comment by L0Lock** [comment]
Same for me, still crashing sometimes.

**Comment by DotBow** [comment]
Hi!
Thanks for testing, will dig in more...

**Comment by DotBow** [comment]
It started to happen for me too when I started to use 3.0 regularly.
Can anyone confirm that?

**Comment by Emby01** [comment]
I've been trying to download Blender 3.3.1 (LTS) through the Blender Launcher, but long before the download finishes, the application just crashes, and the download gets cancelled.

I've tried downloading it three times now and it crashes before the download can finish, every time

I'm using Blender Launcher 1.15.1 btw

**Comment by xan2622** [comment]
Hi @Emby01.
Does it still crash with the latest _development version_ of Blender Launcher ? 👉 [link here](https://github.com/DotBow/Blender-Launcher/discussions/163#discussioncomment-3530477)

This might be your last chance to get Blender Launcher work since DotBow (its author) has recently mentioned that the development is halted: https://github.com/DotBow/Blender-Launcher/discussions/188

**Comment by Emby01** [comment]
@xan2622 
Nope, it just crashed again before it could complete its download. It got to around 116MB out of ~270MB of the download.
